src/CBT.Core/Tasks/WriteItemToJsonFile.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using CBT.Core.Internal;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+
+namespace CBT.Core.Tasks
+{
+    public class WriteAssetsFlagJsonFile : Task
+    {
+
+        [Required]
+        public ITaskItem[] Input { get; set; }
+
+        [Required]
+        public string File { get; set; }
+
+        public override bool Execute()
+        {
+            if (string.IsNullOrWhiteSpace(File))
```

Comment:
Won't the `[Required]` attribute on `File` guarantee that its set so you don't have to verify it has a value?